editor/components/EditableField.tsx

```diff
@@ -55,8 +60,15 @@ export default function EditableField({
     },
   });
 
+  // Sets the message to be shown if there was an error in the graphql mutation
+  const [mutationError, setMutationError] = useState<string | null>(null);
+
   const onSubmit = async (data: FormValues) => {
-    await onSave(data.value);
+    await onSave(data.value).catch((error) => {
+      setMutationError(
+        getMutationErrorMessage ? getMutationErrorMessage(error) : null
```

Comment:
I found out that this `onSave` only runs if there are no validation errors from RHF, so this wont run for example if you type in `70a` to the speed limit col in the crash details page, RHF validates the input and catches the error before handling the `onSave`. This will only run if there are no validation errors, and then it will catch any graphql mutation errors to show as error feedback in the form.

Reply:
Niiiice!

Reply:
What do you think about adding a fallback/default error message when `getMutationErrorMessage` is null? Like instead of `null` we use the generic `Something went wrong message`? That _should_ give us error feedback across the whole app ✨ 

Edit: I did a quick test of this and we'll need a few minor tweaks to make that work universally. I'll open a follow-up issue.